Remove course about 'Overview' link/tab.

mtyaka opened the pull request:
It looks like in the past there used to be several tabs on the course about page, but the other tabs have been commented out years ago. Having a single tab is pointless and confusing, so this PR removes it.

![screen shot 2016-02-15 at 08 38 14](https://cloud.githubusercontent.com/assets/32585/13042470/b8100af2-d3c0-11e5-941a-6c1dd4619752.png)

This PR removes the single tab:

![screen shot 2016-02-15 at 08 41 04](https://cloud.githubusercontent.com/assets/32585/13042493/c95bca9e-d3c0-11e5-9499-f2551f75f802.png)


**Partner information**: 3rd party-hosted open edX instance
**JIRA ticket**: https://openedx.atlassian.net/browse/OSPR-1146
**LMS Sandbox**: http://pr11547.sandbox.opencraft.com/
**Studio Sandbox**: http://studio.pr11547.sandbox.opencraft.com/

edx-webhook commented:
Thanks for the pull request, @mtyaka! It looks like you're a member of a company that does contract work for edX. If you're doing this work as part of a paid contract with edX, you should talk to edX about who will review this pull request. If this work is not part of a paid contract with edX, then you should ensure that there is an OSPR issue to track this work in [JIRA](https://openedx.atlassian.net), so that we don't lose track of your pull request.

To automatically create an OSPR issue for this pull request, just visit this link: https://openedx-webhooks.herokuapp.com/github/process_pr?repo=edx%2Fedx-platform&number=11547

edx-webhook commented:
Thanks for the pull request, @mtyaka! I've created [OSPR-1146](https://openedx.atlassian.net/browse/OSPR-1146) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).

itsjeyd commented:
:+1:

cahrens commented:
Is there any associated sass that should be deleted?

cahrens commented:
TNL reviewed this in triage, we are :+1: but would like the author to spend a short amount of time searching for the associated sass and/or JavaScript. It's not clear if those elements still exist in the codebase.

mtyaka commented:
Thanks for the review @cahrens, and good idea about removing the associated CSS - I found some CSS that wasn't being used anywhere else and removed it. I also set up a sandbox and added links to the PR description.

cahrens commented:
Thanks-- looks good! Go ahead and merge.
